docs/test-page/index.md

```diff
@@ -0,0 +1,362 @@
+      </div>
+      <div class="col-12 col-lg-3">
+        <button type="button" class="btn btn-ui btn-sm mt-8" data-toggle="popover" data-placement="top" title="Popover title" data-content="And here's some amazing content. It's very engaging. Right?">
+          <span class="fas fa-question" aria-hidden="true"></span>
+          <span class="sr-only visually-hidden">Click to show popover</span>
+        </button>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">
+        <hr class="minor">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12 col-lg-4">
+        {%- include 'markup/checkboxes.njk' -%}
+      </div>
+      <div class="col-12 col-lg-4">
+        {%- include 'markup/radios.njk' -%}
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">
+        <hr class="minor">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12 col-lg-4">
+        {%- include 'markup/toggle-tokens-exclusive.njk' -%}
+      </div>
+      <div class="col-12 col-lg-4">
+        {%- include 'markup/toggle-tokens-inclusive.njk' -%}
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">
+        <hr class="minor">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12 col-lg-3">
+        <div class="form-group">
+          <label for="textarea1" class="label">Textarea</label>
+          <textarea class="form-control" id="textarea1" rows="5"></textarea>
+          <small>This accepts any kind of alphanumeric characters.</small>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- table -->
+
+<div class="content-container">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-12">
+        <h2 class="legend">Heading</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12 col-lg-3">
+        {%- include 'markup/input-text.njk' -%}
+      </div>
+      <div class="col-12 col-lg-3">
+        {%- include 'markup/select.njk' -%}
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">
+        {%- include 'markup/sample-table.njk' -%}
+      </div>
+    </div>
+    <div class="row mb-n6">
+      <div class="col-12">
+        <hr class="minor">
+      </div>
+    </div>
+    {%- include 'markup/pagination.njk' -%}
+  </div>
+</div>
+
+<hr class="major">
+
+<!-- typography and toc -->
+
+<div class="content-container">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-12 col-lg-4 order-lg-2">
+        {%- include 'markup/sample-toc.njk' -%}
+      </div>
+      <div class="col-12 col-lg-8 order-lg-1">
+        {%- include 'markup/sample-typography.njk' -%}
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">
+        <hr class="minor">
+      </div>
+    </div>
+    {%- include 'markup/button-page-actions.njk' -%}
+  </div>
+</div>
+
+<!-- modals and toasts buttons -->
+
+<div class="content-container">
```

Comment:
If this were a btn-container, the buttons would space better at mobile sizes. The specificity of the button spacing needs both the .btn-container and the .btn classes.
```suggestion
<div class="btn-container">
```

Reply:
Not worried about it here